tools/create_class_definitions.py

```diff
@@ -1,24 +1,47 @@
+import argparse
 import collections
 import copy
 import json
 from pathlib import Path
 
 import construct
 
+parser = argparse.ArgumentParser()
+def game_argument_type(s: str) -> str:
+    if s not in ["dread", "sr"]:
+        raise ValueError(f"No enum named {s} found")
+    return s
+
+parser.add_argument("game", help="The game to create the class definitions for",
+                     type=game_argument_type, choices=list(["dread", "sr"]))
```

Comment:
`list([ ... ])` doesn't cause a ruff error, I need to add more rules to this repo :P